src/NUnitConsole/nunit3-console.tests/CosoleRunnerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using NUnit.Common;
+using NUnit.Engine;
+using NUnit.Engine.Extensibility;
+using NUnit.Engine.Services;
+using NUnit.Framework;
+
+namespace NUnit.ConsoleRunner.Tests
+{
+    class CosoleRunnerTests
+    {
+        [Test]
+        public void XXX()
+        {
+            var testEngine = new TestEngine();
+            testEngine.Services.Add(new TestResultService());
+            var consoleRunner = new ConsoleRunner(testEngine, new ConsoleOptions("mock-assembly.dll", "--work=c:\\windows\\"), new ColorConsoleWriter());
+            Assert.Throws<NUnitEngineException>(() => { consoleRunner.Execute(); });
```

Comment:
I would like to check the message in the exception. You can check that the message contains a substring.